src/main/java/org/opengis/cite/gmljpx20/core/CoreTests.java

```diff
@@ -109,7 +109,7 @@ public void containsGmlCoverageDescriptions() {
             XMLBox auxXmlBox = (XMLBox) xmlBox;
             Document doc = docBuilder.parse(new InputSource(new StringReader(auxXmlBox.xmldata.trim())));
             // Note: Just check doc element for allowed coverage types?
-            Boolean hasGmlCovElems = (Boolean) XMLUtils.evaluateXPath(doc, "//cov:*",
+            Boolean hasGmlCovElems = (Boolean) XMLUtils.evaluateXPath(doc, "//*[local-name()=’cov’]",
```

Comment:
What should be tested here? Is it really the local-name? Or the namespace url? Then the function `namespace-uri()` should be used.